src/docs/asciidoc/reference_doc/ui_api.adoc

```diff
@@ -247,10 +248,280 @@ It can be used by a template to launch some processing when loading is complete
 
+For example : 
+
+```
+        opfab.currentCard.registerFunctionToGetUserResponse ( () => 
+          {
+                const question = document.getElementById('question').value;
+
+                if (question.length <1) return {
+                    valid: false,
+                    errorMsg : "You must provide a question"
+                }
+            
+                const card = {
+                    summary: { key: "question.summary" },
+                    title: { key: "question.title" },
+                    severity: "ACTION",
+                    data: {
+                        question: question,
+                    }
+                };
+                return {
+                    valid: true,
+                    card: card,
+                    viewCardInCalendar: false
+                };
+
+            })
+```
+
+== CurrentUserCard API
+
+
+=== Function getEditionMode
+
+The template can know if the user is creating a new card or editing an existing card by calling the _opfab.currentUserCard.getEditionMode()_ function. The function will return one of the following values:
+
+* 'CREATE'
+* 'EDITION'
+
+```
+        const mode = opfab.currentUserCard.getEditionMode();
+```
+
+=== Function getEndDate
+
+The template can know the current endDate of the card in creation or edition by calling the _opfab.currentUserCard.getEndDate()_ function. The function will return a number corresponding to the endDate as epoch date value.
+
+```
+        const endDate = opfab.currentUserCard.getEndDate();
+```
+
+=== Function getExpirationDate
+
+The template can know the current expirationDate of the card in creation or edition by calling the _opfab.currentUserCard.getExpirationDate()_ function. The function will return a number corresponding to the expirationDate as epoch date value.
+
+```
+        const expirationDate = opfab.currentUserCard.getExpirationDate();
+```
+
+
+=== Function getLttd
+
+The template can know the current lttd of the card in creation or edition by calling the _opfab.currentUserCard.getLttd()_ function. The function will return a number corresponding to the lttd as epoch date value.
+
+```
+        const lttd = opfab.currentUserCard.getLttd();
+```
+
+=== Function getProcessId
+
+The template can know the process id of the card by calling the _opfab.currentUserCard.getProcessId()_  function. The function will return a string corresponding to the process id.
+
 ```
-        opfab.currentCard.registerFunctionToGetUserResponse
+        const id = opfab.currentUserCard.getProcessId();
 ```
+=== Function getSelectedEntityRecipients
 
+The template can know the list of entities selected by the users as recipients of the card by calling the _opfab.currentUserCard.getSelectedEntityRecipients()_ function.  The function will return an array of entity ids.
```

Comment:
"selected by the users" -> "selected by the user"       singular 
"function.  The function" -> "function. The function"   one too many spaces